<?xml version="1.0"?>
<launch>

	<arg name="car_id" default="0" />

    <node pkg="multi_car_localization" type="super_fake_lidar.py" name="uwb" output="screen">
		<param name="car_id" value="$(arg car_id)" />
    </node>
</launch>
